test/associations/oneToOne/camelize.test.js

```diff
@@ -1,9 +1,10 @@
+"use strict";
+
 var it = require('it'),
     assert = require('assert'),
     helper = require("../../data/oneToOne.helper.js"),
     patio = require("index"),
     comb = require("comb"),
-    Promise = comb.Promise,
     hitch = comb.hitch;
```

Comment:
I was removing these as jshint complained about them. My guess is no, but I could have missed it. I'll check it out next time I'm working on patio.